site/source/docs/porting/connecting_cpp_and_javascript/Interacting-with-code.rst

```diff
@@ -220,6 +220,11 @@ The types of the parameters you pass to functions need to make sense.
 Integers and floating point values can be passed as is. Pointers are
 simply integers in the generated code.
 
+.. note:: Since ``long long`` variables are 64 bits long, they can not be
+   represented in JavaScript. For this reason if function in C/C++ code receives
+   ``long long`` parameter, Emscripten will emit 2 parameters to represent it,
+   for lower and higher 32 bits accordingly.
```

Comment:
I haven't tested myself (sorry), but this makes it sound as if you can't actually call a function with 64 parameters with `ccall` or `cwrap`. I'd remove the last sentence, as what Emscripten does for internal calls is not really relevant to this part of the docs. Instead it would help to give an example of how it is done. preamble.js.rst already mentions that it needs two parameters, but it could do with an example.

Reply:
To be honest, I did not test this behaviour with `call` / `cwrap`. The issue was - I tried to build library and export functions from it, so when I did it - it required me to actually pass 2 parameters per function instead of one. This I try explain here in this part of the doc, so if someone stumbles upon it, they know what to do.

Reply:
Actually the second sentence would be fine, if you just add a couple of words: "emit 2 parameters" -> "emit functions with 2 parameters". I think that's probably what you meant, just a bit clearer.

An example showing what you've done would be fine, and if needed we can always add extra `ccall` specific examples later.

Reply:
I agree with @curiousdannii , the text would be better with adding `the function` there.